### PR TITLE
Make sure to produce placeholders…

### PR DESCRIPTION
**Pull request by rklaehn**
even when skipping purged leafs or branches

### CODE REVIEW

File: banyan/src/forest/read.rs [hunk 1]
```diff
@@ -144,7 +144,9 @@ where
     }
 
     pub(crate) fn get0(&self, index: &Index<T>, mut offset: u64) -> Result<Option<(T::Key, V)>> {
-        assert!(offset < index.count());
+        if offset >= index.count() {
```
Comment (rklaehn) [inline banyan/src/forest/read.rs:147]: This is an unrelated change. better to return None than to panic.

**Review by wngr — APPROVED** [review]
